### DISCUSSION TITLE
Model update in admin

### DISCUSSION THREAD

**Discussion by florian-krueger** [strapi/strapi#8808] [Legacy Topics]
Hi all,

I am adding some infos to entities in the model lifecycle, for example i add geo coordinations based on a provided street address. Since the admin now stays on the entity-view after save (which is a great thing), I have the problem that updates on the model are not visible in the admin ui after save. Is there any workaround or I am doing something wrong?

Thanks,
Florian

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
